lib/open_project/text_formatting.rb

```diff
@@ -418,5 +425,19 @@ def replace_toc(text, headings)
         end
       end
     end
+
+    #
+    # displays the current url plus an optional anchor
+    #
+    def full_url(anchor_name = '')
+      return false if current_request.nil?
```

Comment:
Old habit. I personally dislike some constructs along the lines of "hey, `nil` is falsy". I can change it, though.